src/services/schema-export/tests/models/all/cs/RequiredTypes.cs

```diff
@@ -32,6 +32,14 @@ public class RequiredTypes : RealmObject
         [MapTo("dataRequired")]
         public byte[] DataRequired { get; set; }
 
+        [Required]
+        [MapTo("objectIdRequired")]
+        public ObjectId ObjectIdRequired { get; set; }
```

Comment:
When a type contains `ObjectId` or `Decimal128`, the file needs to have a `using MongoDB.Bson;` directive at the top.